src/main/app/src/Components/Landscape/Modals/Item/Item.tsx

```diff
@@ -136,23 +145,28 @@ const Item: React.FC<Props> = ({ useItem, locateItem, fullyQualifiedItemIdentifi
         onClick={() => {
           locateItem(item.fullyQualifiedIdentifier);
         }}
-        className={classes.floatingButton}
       >
         <FilterCenterFocus />
       </IconButton>
     ) : null;
 
   const labels = item ? getLabels(item) : null;
   const extend = small ? (
-    <IconButton onClick={() => setCompact(!compact)} className={classes.floatingButton}>
+    <IconButton onClick={() => setCompact(!compact)}>
       <MoreVertSharp />
     </IconButton>
   ) : null;
   return (
     <Card className={classes.card}>
       <CardHeader
         title={item ? item.name || item.identifier : null}
-        avatar={item ? <img src={getItemIcon(item)} alt='Icon' className={classes.icon} /> : ''}
+        avatar={
+          item ? <Avatar
+              imgProps={{ style: { objectFit: 'contain' } }}
+              src={getItemIcon(item)}
+              style={{ backgroundColor: 'white', border: '2px solid #' + item.color }}
```

Comment:
How about: `style={{ backgroundColor: 'rgba(255, 255, 255, 0.5)', border: '2px solid #' + item.color }}`?

This makes it transparent white, instead of just white. In case of the redis icon (which contains white text), it is not well readable.